gatherpress.php

```diff
@@ -1,22 +1,54 @@
 <?php
 /**
- * Plugin Name: GatherPress
- * Plugin URI:  https://gatherpress.org/
- * Description: GatherPress adds event management and more to WordPress.
- * Author:      The GatherPress Community
- * Author URI:  https://gatherpess.org/
- * Version:     0.4
- * Text Domain: gatherpress
- * License:     GPLv2 or later (license.txt)
+ * Plugin Name:         GatherPress
+ * Plugin URI:          https://gatherpress.org/
+ * Description:         GatherPress adds event management and more to WordPress.
+ * Author:              The GatherPress Community
+ * Author URI:          https://gatherpess.org/
+ * Version:             0.4
+ * Minimum PHP Version: 7.3
+ * Text Domain:         gatherpress
+ * License:             GPLv2 or later (license.txt)
  *
  * @package GatherPress
  */
 
 // Constants.
+define( 'GATHERPRESS_VERSION', current( get_file_data( __FILE__, array( 'Version' ), 'plugin' ) ) );
+define( 'GATHERPRESS_MINIMUM_PHP_VERSION', current( get_file_data( __FILE__, array( 'Minimum PHP Version' ), 'plugin' ) ) );
 define( 'GATHERPRESS_CORE_PATH', __DIR__ );
 define( 'GATHERPRESS_CORE_URL', plugin_dir_url( __FILE__ ) );
-define( 'GATHERPRESS_VERSION', current( get_file_data( __FILE__, array( 'Version' ), 'plugin' ) ) );
 define( 'GATHERPRESS_REST_NAMESPACE', 'gatherpress/v1' );
 
+/**
+ * Check version of PHP before loading plugin.
+ */
+if ( version_compare( PHP_VERSION_ID, GATHERPRESS_MINIMUM_PHP_VERSION, '<' ) ) {
```

Comment:
Change is unrelated to GP-99, but wanted to add a PHP version check because our code requires higher than WP minimum requirement.